src/components/PDFView/index.native.js

```diff
@@ -164,7 +164,7 @@ function PDFView({onToggleKeyboard, onLoadComplete, fileName, onPress, isFocused
     return onPress && !successToLoadPDF ? (
         <PressableWithoutFeedback
             onPress={onPress}
-            style={[themeStyles.flex1, themeStyles.flexRow, themeStyles.alignSelfStretch]}
+            style={[themeStyles.flex1, themeStyles.alignSelfStretch]}
```

Comment:
You should remove `flexRow` conditionally like I suggested in my latest proposal:

```js
style={[themeStyles.flex1, themeStyles.alignSelfStretch, !failedToLoadPDF && themeStyles.flexRow]}
```
this way we avoid any possible regressions since the Pressable children can also render the PDF itself or the PDFPasswordForm, not only the View containing the error.

Reviewer already agreed with this method, see https://github.com/Expensify/App/issues/35808#issuecomment-1953615706.

cc @getusha

Reply:
thanks for raising this @ikevin127